schema/samcli.json

```diff
@@ -0,0 +1,338 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema",
+  "title": "AWS SAM CLI samconfig schema",
+  "type": "object",
+  "properties": {
+    "version": {
+      "title": "Config version",
+      "type": "number",
+      "default": 0.1
+    }
+  },
+  "required": [
+    "version"
+  ],
+  "additionalProperties": false,
+  "patternProperties": {
+    "^.+$": {
+      "title": "Environment",
+      "properties": {
+        "init": {
+          "title": "Init command",
+          "description": "",
```

Comment:
Noticed the `init` command description is not picked up. Is that expected?

Reply:
Yup. From what I've found, there is no "help" parameter for the init command. 
Looking a little deeper, though, I can see there are other fields in that area that might be just as good for giving a description. I'll see if I can implement one of them to make sure all description fields are populated 👍